proxy/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "start": "tsx ./index.ts"
+    "start": "tsx ./proxy.ts"
```

Comment:
In all another project the main file is `index.ts` (like in `server/`). Let’s keep name system.

Reply:
Makes sense :-) let's keep it systematical

Reply:
Done!